traits/WhereTrait.php

```diff
@@ -36,17 +36,39 @@ private function initWhereAnd():void {
             }
         }
 
+        /**
+         * Initiates string starter for first call of 'whereAnd' method
+         *
+         * @return void
+         */
         private function initWhereOr():void {
             if ($this->where === "") {
                 $this->where = "WHERE 0";
             }
         }
 
+        /**
+         * Specifies the WHERE statement
+         *
+         * @param string $condition - condition to be added
+         * @return $this
+         */
+        public function where(string $condition = ""):IQueryBuilder {
+            if ($condition === "") {
+                return $this;
+            }
+
+            if ($this->where === "") {
+                $this->where = "WHERE " . $condition;
+            }
+
+            return $this;
+        }
+
         /**
          * Specifies the WHERE ... AND ... statement
          *
          * @param string $condition  - condition to be added
-         * 
```

Comment:
Deleted one line because the new markup standard demands no empty lines between parameters and return types.